pilot/pkg/kube/inject/inject.go

```diff
@@ -474,6 +475,11 @@ func isset(m map[string]string, key string) bool {
 	return ok
 }
 
+func directory(filepath string) string {
+	dir, _ := path.Split(filepath)
```

Comment:
I used `path` instead of `path/filepath` because I happened to see this https://github.com/istio/istio/blob/master/pilot/pkg/model/validation.go#L371